plugins/kubernetes/app/models/kubernetes/api/pod.rb

```diff
@@ -92,11 +92,11 @@ def fetch_logs(container, previous:)
           begin
             timeout_logs do
               @client.watch_pod_log(name, namespace, container: container).each do |log|
-                result << log
+                result << log << "\n"
```

Comment:
for now I'll stay with the 'exact' version ... we can revisit if there is trouble ...